src/test/java/org/prebid/server/auction/ExchangeServiceTest.java

```diff
@@ -1768,6 +1794,11 @@ private static BidRequest givenBidRequest(
         return bidRequestBuilderCustomizer.apply(BidRequest.builder().cur(singletonList("USD")).imp(imp)).build();
     }
 
+    private static BidRequest givenBidRequestWithCur(
```

Comment:
I would argue that creation of this method is justified - it is used only once so `givenBidRequest` could apply here.

Reply:
considering future improvements to currency support, `givenBidRequest` can stand for BidRequest with default currency

Reply:
Let's remove this unused method